app/lib/submission_builder/ty2022/states/ny/individual_return.rb

```diff
@@ -104,10 +104,14 @@ def documents_wrapper
                 xml.dependent do
                   xml.DEP_SSN_NMBR dependent.ssn if dependent.ssn.present?
                   xml.DEP_SEQ_NMBR index+1
-                  xml.DEP_DISAB_IND dependent.eic_disability == true ? 1 : 2
+                  unless dependent.eic_disability_unfilled?
+                    xml.DEP_DISAB_IND dependent.eic_disability_yes? ? 1 : 2
```

Comment:
why do some of them we fill them `2` when its `unfilled` and some of them we just leave it out?

Reply:
That is what NY is requesting.  If the DF data comes out with no associated tag we will store this in the DB as unfilled and not show it on the state XML.  Whenever it is present as true or false though we will show it with the appropriate code 1 or false respectively.